comments/stakeholder_annotations.md

```diff
@@ -0,0 +1,46 @@
+## Objective 
+
+Direct users through an SMP-native painpoint/stakeholder annotation process that allows them to respond to map content. The input data generates an output table with the following structure, accessed via API:
+
+| Participant | Object | Category | Comment |
+|-------------|--------|----------|---------|
+| Joe Bloggs | Example Painpoint | Surprising | null |
+| Joe Bloggs | Example Stakeholder | Question | "blahblahblah" |
+...
+
+This may then be aggregated into an equivalent data format as the table in the existing Powerpoint doc.
+
+## Process
+
+### User identity
+
+A user is authorised to access a map by including their email address (or a wildcarded version of it) in the whitelist for the map's project.
+
+The user is emailed a link to the map. When they access it, they are first prompted to enter their email email. This is checked against the whitelist for access, and will also be used as their identity for entering annotations.
+
+### Explaining the exercise to user
+
+When the user accesses the map for the first time, they are greeted with an Introduction splash outlining the exercise and explaining how to leave annotations (categories and comments) on painpoints/stakeholders.
+
+### Leaving annotations
+
+Clicking on a painpoint brings up the annotation interface. A user can assign a category to a painpoint (Interesting/Surprising etc.) and optionally include a comment. (Emptystring if not entered?)
```

Comment:
I think we can probably do better than this, but that's for later-discussion I think.